client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -461,7 +461,7 @@ export const formatSites = ( sites: Array< Site > = [] ): Array< SiteData > | []
 			site: {
 				value: site,
 				error: ! site.is_connection_healthy,
-				status: '',
+				status: 'active',
```

Comment:
Setting `active` as default. This is not used anywhere. I just don't want to keep it `''`